CHANGELOG.md

```diff
@@ -5,12 +5,14 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
-- Ensure map-matching considers HOV-only roads as auto accessible. [#6785](https://github.com/mapbox/mapbox-navigation-android/pull/6785)
 
 ## Mapbox Navigation SDK 2.9.7 - 27 January, 2023
 ### Changelog
 [Changes between v2.9.6 and v2.9.7](https://github.com/mapbox/mapbox-navigation-android/compare/v2.9.6...v2.9.7)
 
+#### Bug fixes and improvements
+- Fixed some internal Copilot-related issues and added minor improvements.
```

Comment:
PR link?

Reply:
There were some but opted to not link as they were internal.